starter-code/functions-and-arrays.js

```diff
@@ -1,14 +1,27 @@
 // Find the maximum
 function maxOfTwoNumbers (first, second) {
-
+  if (first>second) {
+     return first;
+  } else {
+    return second;
+  }
```

Comment:
You could do something fancier like:

function maxOfTwoNumbers (first, second) {
  return first > second ? first : second;
}